lib/scrapPaperReader.js

```diff
@@ -0,0 +1,21 @@
+const config = require('./config');
+const path = require('path');
```

Comment:
I removed most of the code in this file for part-1 as it is distracting and confusing. You will see a much larger file here in part-2.